test/lib.bash

```diff
@@ -92,6 +93,9 @@ function downstream_serving_e2e_tests {
   return $failed
 }
 
+function downstream_knative_kafka_e2e_tests {
+}
```

Comment:
This causes a syntax error (which subsequently causes the "function not found" error).

See

```
 ./test/e2e-tests.sh
./test/lib.bash: line 97: syntax error near unexpected token `}'
./test/lib.bash: line 97: `}' 
```